starboard/linux/x64x11/gcc/6.3/toolchain/BUILD.gn

```diff
@@ -12,11 +12,19 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import("//build/toolchain/gcc_toolchain.gni")
+import("//starboard/build/toolchain/cobalt_toolchains.gni")
 import("//starboard/shared/toolchain/overridable_gcc_toolchain.gni")
 
 # Directory for GCC 6.3 if it is installed as a system dependency.
 _default_gcc_6_3_bin_dir = "/usr/bin"
 
+cobalt_clang_toolchain("cobalt") {
+  variables = {
+    native_linker_path = "${_default_gcc_6_3_bin_dir}/clang++"
+  }
+}
+
 overridable_gcc_toolchain("target") {
```

Comment:
I forgot to say for clang-3.9, but I think because we're full switching over the platform we can remove this toolchain entirely (unless we want to keep the monolithic build around for some reason)

Reply:
ok, I can take it out.